recipes/sensormapgeo/meta.yaml

```diff
@@ -0,0 +1,50 @@
+{% set name = "sensormapgeo" %}
+{% set version = "0.4.5" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/sensormapgeo-{{ version }}.tar.gz
+  sha256: 4f435416cb98602882e0a122a5499b8ffb027a36b6eae5529e493ae235a5cfcd
+
+build:
+  number: 0
+  skip: true  # [py<36]
```

Comment:
Looks like this can be a `noarch` package. You can remove the skip, add `noarch: python` here, and add `python <3.6` in host and run below.

Reply:
python >3.6 in the host section, right? I don`t want to build for the old Python versions.

Reply:
If you do `noarch` it should not matter. But I guess it does not hurt too.

Reply:
> Looks like this can be a `noarch` package. 

Sure? There is a platform specific requirement:

`   - _openmp_mutex =* =*_llvm  # [linux]`

Reply:
Yep. Missed that one :-(

Reply:
Ok, so I keep it as it is. Does this also mean that all packages that depend on sensormapgeo cannot be noarch packages, too?